app/javascript/components/NavBar.vue

```diff
@@ -30,7 +30,7 @@
 
 <script>
 import logo from 'images/logo.png'
-import DeleteButton from 'components/DeleteButton'
+import DeleteButton from './DeleteButton'
```

Comment:
Just curious -- why is this needed/available now?

Reply:
It was not part of the other changes here. I just noticed that `NavBar` and `DeleteButton` are siblings, so this is simpler and helps IDEs 'go-to' the file (they often can't figure out where `components/` is).